base/data/gm4/tags/blocks/foliage.json

```diff
@@ -0,0 +1,5 @@
+{
+  "values": [
+    "#minecraft:flowers"
```

Comment:
Kinda of seems irrelevant to have a tag with only one tag. 
Should dead_bush, fern, large fern, grass, tall grass be added here? IDK the implications of this; the blocks I listed are foliage, but don't have the same behavior as flowers (flowers can't be replaced when placing a block).

Reply:
Perhaps adding the mushrooms would be viable? Unlike the blocks listed in the other comment, mushrooms don't get replaced when placing a block.